kotest-common/src/commonMain/kotlin/io/kotest/mpp/Reflection.kt

```diff
@@ -1,5 +1,6 @@
 package io.kotest.mpp
 
+import kotlin.reflect.KCallable
```

Comment:
You're only using this in the jvm implementation right, so you could remove the import ?